src/SFA.DAS.PensionsRegulator.Database/StoredProcedures/GetOrganisationsByPAYEReference.sql

```diff
@@ -1,5 +1,5 @@
 ﻿CREATE PROCEDURE [dbo].GetOrganisationsByPAYEReference
-	@PAYESchemeReference nvarchar(50)
+	@PAYESchemeReference varchar(25)
```

Comment:
will it ever be this long?

Reply:
I am taking the data types from the DB design doc, so just doing as I'm told here :)